kajas/src/app/settings/settings.component.html

```diff
@@ -0,0 +1,67 @@
+<app-header></app-header>
+<div class="settings-container">
+    <div class="modal">
+        <div class="modal-content">
+            <span class="close">&times;</span>
+            <div class="left-column">
+                <h1>Settings</h1>
+                <ul class="tabs">
+                    <li 
+                        class="tab-link" 
+                        (click)="selectTab('change-email')">
+                        Change Email
+                    </li>
+
+                    <li 
+                        class="tab-link" 
+                        (click)="selectTab('change-password')">
+                        Change Password
+                    </li>
+
+                    <li 
+                        class="tab-link" 
+                        (click)="selectTab('help-and-support')">
+                        Help & Support
+                    </li>
+
+                    <li 
+                        class="tab-link" 
+                        (click)="selectTab('terms-conditions')">
+                        Terms & Conditions
+                    </li>
+
+                    <li 
+                        class="tab-link" 
+                        (click)="selectTab('privacy-policy')">
+                        Privacy Policy
+                    </li>
+
+                    <li 
+                        class="tab-link" 
+                        (click)="selectTab('deactivate-account')">
+                        Deactivate Account
+                    </li>
+                </ul>
+            </div>
+            <hr class="divider">
+            <div class="right-column">
+                <app-change-email 
+                    *ngIf="selectedTab === 'change-email'">
+                </app-change-email>
+
+                <app-change-password 
+                    *ngIf="selectedTab === 'change-password'">
+                </app-change-password>
+
+                <app-help-and-support
+                *ngIf="selectedTab === 'help-and-support'">
```

Comment:
### Use the built-in control flow instead of structural directive.
[https://docs.google.com/presentation/d/1Mdcq3WcHksBlbCvxi6wx5vajn4v8LGlJRnBIu3x4fvw/edit#slide=id.g26ff0f47168_0_1](url)

### For template control flow, observe proper indentation.
[https://docs.google.com/presentation/d/1Mdcq3WcHksBlbCvxi6wx5vajn4v8LGlJRnBIu3x4fvw/edit#slide=id.g270ee9174f4_0_15](url)